Code-Sleep-Python/Link Shortener/source-code.py

```diff
@@ -0,0 +1,8 @@
+import pyshorteners
+
+link = input("enter the link:") #variable
+shortener = pyshorteners.Shortener() #class object
+
+x = shortener.tinyurl.short(link) #shorting the link
+
+print(x)
```

Comment:
also modify it as a cli-comand or a bash instruction which people can use at command line as a CLI tool itself